core/modules/system/js/modules.js

```diff
@@ -92,6 +100,8 @@ Backdrop.behaviors.moduleFilterByText = {
 
       // @todo Use autofocus attribute when possible.
       $input.focus().on('keyup', filterModuleList);
+      $('select[data-filter-tags]').change(filterModuleList);
+      $('input[data-filter-source]').change(filterModuleList);
```

Comment:
This looks like cruft from a previous version as well.

Reply:
fixed